docs/content/en/docs/clustermgmt/security/certificate-management/script-renew-certs.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Manually Renew Cluster Certificates"
+linkTitle: "Script To Renew Certificates"
```

Comment:
I am curious if these instructions should just be in the README of the repo and we can just refer the readme in the official docs. But I would defer to docs writer on that

Reply:
I'm okay with having it here. But I don't have a strong opinion.